README update for sequelize v6

lukashroch opened the pull request:
Update readme

- how to upgrade to sequelize v6 & sequelize-typescript v2
- outline breaking changes
- sequelize-typescript v1 with v5

intellix commented on README.md:
```diff
@@ -37,29 +38,96 @@ Decorators and some other features for sequelize (v5).
+  name: string;
+}
+
+interface PersonCreationAttributes {
+  name: string;
+}
```
```suggestion
interface PersonCreationAttributes extends Omit<PersonAttributes, 'id'> {
}
```

lukashroch replied:
Thanks @intellix, added with corresponding import statement. Good idea to keep it consistent with sequelize official TS guide.
...though anyone can freely to construct the interfaces however feels appropriate.

DaPfeifa commented:
I hope this can be merged soon.
Otherwise you always run into problems when migrating to Sequelize v6.

lukashroch commented:
I've squashed the changes and added a note about current state of implementation, so it's more clear what currently works.

- less strict sequelize V6 works quite fine - this is equivalent to what we had so far
- more strict sequelize introduced in v6.2+ is not yet implemented, will need more work

@RobinBuschmann if this could be merged, it would be much appreciated. I should have done the README update along with the code change, so users knew current state and what needs to be done. Thanks!
